JavaScript/Post_Tweets/index.js

```diff
@@ -0,0 +1,38 @@
+var Twit = require('twit')
+const fs = require('fs');
+var T = new Twit({
+  consumer_key:         'YOUR_CONSUMER_KEY',
+  consumer_secret:      'YOUR_CONSUMER_SECRET',
+  access_token:         'YOUR_ACCESS_TOKEN',
+  access_token_secret:  'YOUR_ACCESS_SECRET',
+})
+ 
+// post tweet 
+var text = "ENTER_TWEET_TEXT"
+T.post('statuses/update', { status: text }, function(err, data, response) {
+  console.log("tweeted", text)
+})
+ 
+//
+// post a tweet with media
+//
+var imagePath ='ENTER_IMAGE_PATH'
+var caption = 'ENTER_CAPTION'
+
```

Comment:
Same for them as well. Make it dynamic instead of asking Users to change the Code to make it run.